cmd/image/qcow2ova/prep/templates.go

```diff
@@ -68,7 +68,7 @@ done
 grub2-mkconfig -o /boot/grub2/grub.cfg
 rm -rf /etc/sysconfig/network-scripts/ifcfg-eth0
 {{if .RootPasswd }}
-echo {{ .RootPasswd }} | passwd root --stdin
+echo root:{{ .RootPasswd }} | chpasswd
```

Comment:
Is there any doc reference we know which talks about `chpasswd` command fixes the issue we are facing? Its worth documenting the information why this change is fixing the issue we are facing